spec.md

```diff
@@ -119,12 +107,31 @@ The propagation and trace-joining methods come in two flavors: binary and text.
 - The *text* format is a platform-idiomatic map from (unicode) `string` to `string`; it is better-suited to pretty-printing and debugging
 - The *binary* format is an opaque byte array and is better-suited to compact, high-performance encoding, decoding, and transmission
 
-Note that there is no expectation that different tracing systems propagate `Spans` in compatible ways. Though OpenTracing is agnostic about the tracing implementation, for successful inter-process handoff it's essential that the processes on both sides of a propagation use the same tracing implementation.
+Note that there is no expectation that different tracing systems propagate `TraceContext` in compatible ways. Though OpenTracing is agnostic about the tracing implementation, for successful inter-process handoff it's essential that the processes on both sides of a propagation use the same tracing implementation.
+
+The `TraceContext` interface must have the following capabilities:
+- Propagating a TraceContext instance by encoding it as either...
```

Comment:
Yeah, I thought it was an extraneous detail for the instrumenter to have to worry about propagating both separately.

Also, in the long run, where people are hopefully using a corpus of common instrumentation, Attributes would probably be added by an "end-user", not the instrumenter who is doing the RPC transport instrumentation originally.  So the user who cares about Attribute propagation would not have access here anyway.